cfglib.py

```diff
@@ -105,6 +105,10 @@
                help='Size of one chunk to transfer via SSH'),
     cfg.StrOpt('group_file_path',
                help='Path to file with the groups of VMs'),
+    cfg.BoolOpt('all_vms', default=False,
+                help="Migrate all VM's from all tenants. User, specified in "
```

Comment:
No, we do not check it. But we have it in our requirements (https://github.com/MirantisWorkloadMobility/CloudFerry/blob/devel/README.md):
`Credentials of global cloud admin for both clouds.`
So we assume, that this user has admin privileges.